websites-css/branching.css

```diff
@@ -0,0 +1,31 @@
+
+.content-section {
+    background-color: var(--color-content-section);
+    padding: 40px 10vh;
+    color: white;
+    
+}
+
+.content-contaioner {
+    margin: auto;
+}
+
+.content-section h2 {
+    font-size: 2.5rem;
```

Comment:
When creating new font-sizes please make them global variables so we all can use them 😄